Adac.cmake

```diff
@@ -0,0 +1,50 @@
+find_program(ARM_ELF_ADAC "${CMAKE_SOURCE_DIR}/adac-hybrid-arm/build/bin/arm-elf-adac")
+
+if(ARM_ELF_ADAC)
+    log_option_enabled("adac")
+
+    set(TOP_DIR "${CMAKE_SOURCE_DIR}/adac-hybrid-arm")
+    set(UTILS_SRC_DIR ${CMAKE_SOURCE_DIR}/adac-hybrid-arm/test/utils)
+    set(BSP_DIR ${CMAKE_SOURCE_DIR}/adac-hybrid-arm/test/bsp)
+    set(ADAC ${CMAKE_SOURCE_DIR}/adac-hybrid-arm/build/bin/arm-elf-adac)
+
+    set(COMMON_ADAC_CFLAGS 
+            -ffunction-sections;
+            -fdata-sections;
+            -march=armv7e-m;
+            -mcpu=cortex-m7;
+            -mfloat-abi=hard;
+            -mfpu=fpv5-d16; 
+            -mlittle-endian;
+            -mthumb )
+
+    set(ADAC_CFLAGS
+            ${COMMON_ADAC_CFLAGS};
+            -Wall;
+            -Wadac,keep_intermediates;
```

Comment:
Keep intermediates is project specific
Paths to includea and utils are project specific
functios-sections are project specific